src/services/email/libs/lookup-lib.js

```diff
@@ -0,0 +1,24 @@
+import { getOsInsightData } from "../libs/os-lib";
+import { getCognitoData } from "../libs/cognito-lib";
+
+export const getLookupValues = async (lookupList, lookupId) => {
+    let returnData = {};
+
+    if (lookupId && lookupList && Array.isArray(lookupList) && lookupList.length > 0) {
+        const lookupPromises = await Promise.allSettled(lookupList.map(async (lookupType) => {
+            switch (lookupType) {
+                case "osInsights":
```

Comment:
Future note, it would be nice if the lookupType was a defined structure or at least enum given its usage across multiple libs/functions. Would just give a bit of type checking.

Reply:
Yeah, now that it doesn't change every commit, can probably look into adding some guard rails :)